test/smoke/testApps/Micrometer/build.gradle.kts

```diff
@@ -4,5 +4,6 @@ plugins {
 
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-web:2.1.7.RELEASE")
-  implementation("io.micrometer:micrometer-core:1.4.1")
+  // TODO (trask) do we still need to support earlier micrometer versions?
+  implementation("io.micrometer:micrometer-core:1.5.0")
```

Comment:
why not 1.8.5?  seemed outdated.

Reply:
Ideally we would test on the lowest version we support and on the latest version (this is what upstream otel instrumentation tests do). I'm not sure what's best in general in our repo. For this particular case I think 1.5.0 is good, especially since it's really not that old in Java years at least (2 years).